addons/mail/static/src/core/web/mail_composer_template_selector.js

```diff
@@ -110,10 +67,21 @@ export class MailComposerTemplateSelector extends Component {
         });
     }
 
+    async onManageTemplateBtnClick() {
+        const action = await this.action.loadAction("mail.action_email_template_tree_all");
+        action.context = {
+            search_default_my_templates: 1,
+            search_default_model_id: this.props.record.data.model_id[0],
```

Comment:
People generally can't read ir.model so would not base searches on it

Reply:
model is stored so you can search on it (would maybe require to add it in search view if not present, does not seem to be the case)

Reply:
Ok, I will search on the field `model`. With that, I don't have to change the fields or worry about the ACL. It's also simpler :smile: